persistent/Database/Persist/Sql/Raw.hs

```diff
@@ -75,9 +75,10 @@ getStmt sql = do
 
 getStmtConn :: SqlBackend -> Text -> IO Statement
 getStmtConn conn sql = do
-    smap <- liftIO $ readIORef $ connStmtMap conn
-    case Map.lookup sql smap of
-        Just stmt -> return stmt
+    let cacheK = mkCacheKeyFromQuery sql
+    mstmt <- liftIO $ statementCacheLookup (connStmtMap conn) cacheK
```

Comment:
```suggestion
    mstmt <- statementCacheLookup (connStmtMap conn) cacheK
```

Technically not a necessary change - wonder why the original has `liftIO` when the code is in `IO` already 🤔